charts/catalog/templates/controller-manager-deployment.yaml

```diff
@@ -34,6 +34,8 @@ spec:
         args:
         - --port
         - "8080"
+        - "--profiling={{ .Values.controllerManager.profiling }}"
```

Comment:
@derekwaynecarr these new flags should be behind a guard, so they don't break the walkthrough. It can be done similarly to how it was done in https://github.com/kubernetes-incubator/service-catalog/pull/904